packages/browser/src/Session.ts

```diff
@@ -333,81 +322,6 @@ export class Session extends EventEmitter {
       return undefined;
     }
 
-    // Unfortunately, regular sessions are lost when the user refreshes the page or opens a new tab.
-    // While we're figuring out the API for a longer-term solution, as a temporary workaround some
-    // *resource* servers set a cookie that keeps the user logged in after authenticated requests,
-    // and expose the fact that they set it on a special endpoint.
-    // After login, we store that fact in LocalStorage. This means that we can now look for that
-    // data, and if present, indicate that the user is already logged in.
-    // Note that there are a lot of edge cases that won't work well with this approach, so it willl
-    // be removed in due time, and is disabled by default.
-    if (
-      options.useEssSession !== true ||
-      options.restorePreviousSession === true
-    ) {
-      window.localStorage.setItem(
```

Comment:
This workaround should not have been used for long enough for this not to be an issue I think. Instead of adding stuff that we'll need to think about removing again, I added a note in the changelog: f4aaab1d